### PR TITLE
Basic gzip compression with express.

### PR DESCRIPTION
**Pull request by meteormanaged**
Added option to README for gzip compression.
Pass along option to worker and express server or default "false".

Just a thought.  Let me know what you think or if there's more work required and I'll take a swing.

### CODE REVIEW

File: lib/express-http-server.js [hunk 1]
```diff
@@ -12,8 +12,12 @@ class ExpressHTTPServer {
     this.username = options.username;
     this.password = options.password;
     this.cache = options.cache;
+    this.gzip = options.gzip || false;
 
     this.app = express();
+    if (options.gzip) {
+      this.app.use(require('compression')());
```
Comment (rwjblue) [inline lib/express-http-server.js]: Need to add `compression` to the `package.json`.

**Comment by rwjblue** [comment]
@meteormanaged - Ping?

**Comment by meteormanaged** [comment]
I'll take care of it now.

**Comment by meteormanaged** [comment]
@rwjblue I believe it should be good, pending CI.  Ping me here or on slack (mikek) if there's any problems.

**Pull request merged**